app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAboutRobot.kt

```diff
@@ -114,7 +209,77 @@ private fun assertLibrariesUsed() {
         .perform(click())
 
     onView(withId(R.id.action_bar)).check(matches(hasDescendant(withText(containsString("Firefox Preview | OSS Libraries")))))
+    goBackButton().click()
 }
 
 private fun goBackButton() =
     onView(CoreMatchers.allOf(withContentDescription("Navigate up")))
+
+class BuildDateAssertion {
+    companion object {
+        // this pattern represents the following date format: "Monday 12/30 @ 6:49 PM"
+        private const val DATE_PATTERN = "EEEE M/d @ h:m a"
+
+        fun isDisplayedDateAccurate(): ViewAssertion {
+            return ViewAssertion { view, noViewFoundException ->
+                if (noViewFoundException != null) throw noViewFoundException
+
+                val textFromView = (view as TextView).text
+                    ?: throw AssertionError("This view is not of type TextView")
+
+                verifyDateIsWithinRange(textFromView.toString(), 1)
```

Comment:
please define a constant for the 1 hour value and comment as to why we need to select a range rather than specific value